libs/novui/src/recipes/tabs.recipe.ts

```diff
@@ -0,0 +1,67 @@
+import { TabsStylesNames } from '@mantine/core';
+import { defineSlotRecipe } from '@pandacss/dev';
+
+const SLOTS: TabsStylesNames[] = ['root', 'list', 'panel', 'tab', 'tabLabel', 'tabSection'];
+
+export const TABS_RECIPE = defineSlotRecipe({
+  className: 'tabs',
+  jsx: ['Tabs'],
+  slots: SLOTS,
+  base: {
+    root: {
+      // default color palette
+      colorPalette: 'mode.cloud',
+      _disabled: {
+        opacity: 'disabled',
+      },
+      '&:hover:not(:disabled)': {
+        opacity: 'hover',
+      },
+    },
+    list: {
+      color: 'typography.text.main',
+      gap: 'margins.layout.tabs.tab-tab !important',
```

Comment:
Why are these `!important` required? Is it due to the old and new stylings living together in the new app?

Reply:
@antonjoel82 , let's circle back on this later. Can you please create a ticket with a detail on why we needed this, and what steps we need to take to fix this.